repo.go

```diff
@@ -0,0 +1,115 @@
+	"ContentPasteIcon":  theme.ContentPasteIcon(),
+	"ContentRedoIcon":   theme.ContentRedoIcon(),
+	"ContentUndoIcon":   theme.ContentUndoIcon(),
+
+	"InfoIcon":     theme.InfoIcon(),
+	"ErrorIcon":    theme.ErrorIcon(),
+	"QuestionIcon": theme.QuestionIcon(),
+	"WarningIcon":  theme.WarningIcon(),
+
+	"DocumentIcon":       theme.DocumentIcon(),
+	"DocumentCreateIcon": theme.DocumentCreateIcon(),
+	"DocumentPrintIcon":  theme.DocumentPrintIcon(),
+	"DocumentSaveIcon":   theme.DocumentSaveIcon(),
+
+	"FileIcon":            theme.FileIcon(),
+	"FileApplicationIcon": theme.FileApplicationIcon(),
+	"FileAudioIcon":       theme.FileAudioIcon(),
+	"FileImageIcon":       theme.FileImageIcon(),
+	"FileTextIcon":        theme.FileTextIcon(),
+	"FileVideoIcon":       theme.FileVideoIcon(),
+	"FolderIcon":          theme.FolderIcon(),
+	"FolderNewIcon":       theme.FolderNewIcon(),
+	"FolderOpenIcon":      theme.FolderOpenIcon(),
+	"ComputerIcon":        theme.ComputerIcon(),
+	"HomeIcon":            theme.HomeIcon(),
+	"HelpIcon":            theme.HelpIcon(),
+	"HistoryIcon":         theme.HistoryIcon(),
+	"SettingsIcon":        theme.SettingsIcon(),
+	"StorageIcon":         theme.StorageIcon(),
+	"DownloadIcon":        theme.DownloadIcon(),
+	// "UploadIcon":          theme.UploadIcon(),
+
+	"ViewFullScreenIcon": theme.ViewFullScreenIcon(),
+	"ViewRestoreIcon":    theme.ViewRestoreIcon(),
+	"ViewRefreshIcon":    theme.ViewRefreshIcon(),
+	"VisibilityIcon":     theme.VisibilityIcon(),
+	"VisibilityOffIcon":  theme.VisibilityOffIcon(),
+	"ZoomFitIcon":        theme.ZoomFitIcon(),
+	"ZoomInIcon":         theme.ZoomInIcon(),
+	"ZoomOutIcon":        theme.ZoomOutIcon(),
+
+	"MoveDownIcon": theme.MoveDownIcon(),
+	"MoveUpIcon":   theme.MoveUpIcon(),
+
+	"NavigateBackIcon": theme.NavigateBackIcon(),
+	"NavigateNextIcon": theme.NavigateNextIcon(),
+
+	"Menu":         theme.MenuIcon(),
+	"MenuExpand":   theme.MenuExpandIcon(),
+	"MenuDropDown": theme.MenuDropDownIcon(),
+	"MenuDropUp":   theme.MenuDropUpIcon(),
+
+	"MailAttachmentIcon": theme.MailAttachmentIcon(),
+	"MailComposeIcon":    theme.MailComposeIcon(),
+	"MailForwardIcon":    theme.MailForwardIcon(),
+	"MailReplyIcon":      theme.MailReplyIcon(),
+	"MailReplyAllIcon":   theme.MailReplyAllIcon(),
+	"MailSendIcon":       theme.MailSendIcon(),
+
+	"MediaFastForward": theme.MediaFastForwardIcon(),
+	"MediaFastRewind":  theme.MediaFastRewindIcon(),
+	"MediaPause":       theme.MediaPauseIcon(),
+	"MediaPlay":        theme.MediaPlayIcon(),
+	// "MediaStop":         theme.MediaStopIcon(),
+	"MediaRecord":       theme.MediaRecordIcon(),
+	"MediaReplay":       theme.MediaReplayIcon(),
+	"MediaSkipNext":     theme.MediaSkipNextIcon(),
+	"MediaSkipPrevious": theme.MediaSkipPreviousIcon(),
+
+	"VolumeDown": theme.VolumeDownIcon(),
+	"VolumeMute": theme.VolumeMuteIcon(),
+	"VolumeUp":   theme.VolumeUpIcon(),
+}
+
+// IconNames Gets the list of the names of the icons available
+var iconNames = make([]string, len(icons))
```

Comment:
You could initialise this here instead of requiring the `main()` func to do it, for example

```
var iconNames = generateRepo()

func generateRepo() []string {
...
```